courses/serializers.py

```diff
@@ -95,9 +95,19 @@ def get_description(self, instance):
 
     def get_courseruns(self, instance):
         """Unexpired and unenrolled course runs"""
-        user = self.context["request"].user
-        active_runs = instance.available_runs(user)
-        return [CourseRunSerializer(instance=run).data for run in active_runs]
+        show_all_runs = self.context.get("show_all_runs", False)
```

Comment:
Nit: can we change this to `all_runs`/`include_all_runs` or something like that? Seems odd to have a parameter asking a serializer to "show" something